api/src/models/master/agreement.js

```diff
@@ -12,8 +12,10 @@ module.exports = require('../../utils/model-schema-generator')(
     write: [{ type: String, trim: true, default: 'sysadmin' }],
 
     recordName: { type: String, default: '' },
-    date: { type: Date, default: Date.now() },
+    recordType: { type: String, default: '' },
+    dateIssued: { type: Date, default: Date.now() },
```

Comment:
We shouldn't bother with Date.now(), better to leave it blank otherwise we are 'creating' things that don't exist.

Reply:
Fixed.  Ill have to go back and update the other models as well then after.